packages/slds-linter/README.md

```diff
@@ -9,66 +13,54 @@
-"setup-lint": "node ./node_modules/@salesforce-ux/sds-linter/build/setup.js"
-```
+Follow the below instructions to integrate SLDS Linter into your project.
 
-once the above scripts are added. You can setup the configuration by running
+---
 
-```
-npm run setup-lint
-```
+### Set Up SLDS Linter in Your Component Repository
 
-which will inturn create `.eslintrc.yml` & `.stylelintrc.yml` in the root folder.
-
-NOTE: If the project root already containing `.eslintrc.yml` & `.stylelintrc.yml`, we don't overwrite those files and the use needs to merge those files manually. You can find the configuration files in this repostory in the root folder.
-
-### Command-Line Interface (CLI)
-
-* To lint all components and styles in your project:
+To install the SLDS Linter Utility in your project, you can use npm:
 
   ```
-  npm run lint:components
-  npm run lint:styles
+    npm install @salesforce-ux/slds-linter --save-dev
   ```
-* To auto-fix some of the most confident fixes automatically.
 
-  ```
-  npm run fix
-  ```
-* To run a report in .sarif format
+To see what all options does slds-linter provide please run `slds-linter --help` which gives the below output.
 
   ```
-  npm run report
+  slds-linter [command]
+
+Commands:
+  slds-linter lint             Run both ESLint and Stylelint
+  slds-linter lint:styles      Run only Stylelint
+  slds-linter lint:components  Run only ESLint
+  slds-linter fix              Fix auto-fixable issues
+  slds-linter report           Generate a linting report
+
+Options:
+  --version  Show version number                                       [boolean]
+  --help     Show help                                                 [boolean]
   ```
 
-### Contribution
+-	slds-linter lint -  Runs the ESlint and Stylelint rules on your HTML/CSS/CMP files and outputs issues.
+-	slds-linter lint:styles - Runs the Stylelint rules on your CSS files and outputs issues.
+-	slds-linter lint:components -  Runs the ESlint rules on your HTML/CMP files and outputs issues.
+-	slds-linter fix: Attempts to automatically fix violations.
+-	slds-linter report: Generates a SARIF report for static analysis.
+
+
+1. Run `slds-linter lint` to see the lint output on terminal. For specific files, you can go ahead with either `slds-linter lint:styles` for lint errors within css files or `slds-linter lint:components` for lint errors within html/cmp files.
+2. To run SLDS Linter, in Terminal, run `slds-linter report` to generate a Sarif report in the project root directory. It will be named as `slds-linter-report.sarif`
+3. Open the generated Sarif file.
+4. Make a note of how many components SLDS Linter has identified that you must update.
+5. Run `slds-linter fix` to automatically fix validation errors in bulk.
 
-We welcome contributions to improve this utility. If you have any suggestions, bug reports, or want to contribute code, please open an issue or pull request.
 
-### License
+## License
```

Comment:
We already have a file named LICENSE.txt at the root. I think this can be removed.